justfile

```diff
@@ -0,0 +1,41 @@
+set quiet
+
+import? '../sdk-codegen/justfile'
+
+_default:
+    just --list --unsorted
+
+install *args:
+    bundle install {{ if is_dependency() == "true" {"--quiet"} else {""} }} {{ args }}
+
+# ⭐ run all unit tests
+test: install
+    bundle exec rake test
+
+# check linting / formatting status of files
+format-check *args: install
+    bundle exec rubocop {{ args }}
+alias lint-check := format-check
```

Comment:
I don't love that formatting and linting are tied together, but it is what it is for now.